### ISSUE TITLE
API Doesn't build w/ Microsoft.Identity.Web >=1.x

### ISSUE THREAD

**Issue by Alfetta159** [Azure-Samples/ms-identity-javascript-react-spa-dotnetcore-webapi-obo#36]
>> Please follow the issue template below. Failure to do so will result in a delay in answering your question.

## Library

- [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
- [x] `@azure/msal-browser@2.4.0`

**Important**: Please fill in your exact version number above, e.g. `msal@1.1.3`.

## Framework
.NET Core 3.1

## Description
API Won't build

## Error Message
'AuthenticationBuilder' does not contain a definition for 'AddMicrosoftIdentityWebApi' and no accessible extension method 'AddMicrosoftIdentityWebApi' accepting a first argument of type 'AuthenticationBuilder' could be found (are you missing a using directive or an assembly reference?)
## Security

- [ ] Is this issue security related?

## Regression

- [ ] Did this behavior work before?

Yes

Version:

## MSAL Configuration

```js
// Provide configuration values here.
// For Azure B2C issues, please include your policies.
```

## Reproduction steps

```js
// Provide relevant code snippets here.
// For Azure B2C issues, please include your policies.
```

## Browsers/Environment

- [x] Chrome
- [ ] Firefox
- [x] Edge
- [ ] Safari
- [ ] IE
- [ ] Other (Please add browser name here)

**Comment by derisen** [comment]
@Alfetta159 I'm unable to reproduce this. Are you sure all the packages are installed? Would you mind sharing your `ProfileAPI.csproj`?

**Comment by jmprieur** [comment]
@Alfetta159 : would it be a long path issue? : you might want to clone the repo close to the root of your hard drive (assuming this windows machine)

**Comment by Alfetta159** [comment]
```
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
    <UserSecretsId>aspnet-ProfileAPI-03230DB1-5145-408C-A48B-BE3DAFC56C30</UserSecretsId>
    <WebProject_DirectoryAccessLevelKey>0</WebProject_DirectoryAccessLevelKey>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.9" />
    <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="3.1.9" Condition="'$(Configuration)' == 'Debug'" />
    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.9" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="3.1.9" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.9" />
    <PackageReference Include="Microsoft.Identity.Web" Version="1.2.0" />
    <PackageReference Include="Microsoft.Identity.Web.MicrosoftGraph" Version="1.2.0" />
  </ItemGroup>

</Project>

```

**Comment by Alfetta159** [comment]
Moved it here: `C:\ms-id-react-spa-webapi-obo` , but no change.

**Comment by Alfetta159** [comment]
A few things got me thinking:

- I found it hard to believe that you guys would publish code that isn't working. 
- @jmprieur 's comment, "assuming that it's a Windows machine" made me try it on a Linux box that I have laying around. Works.
- I saw this same error in other projects of my own.
- Commit d5761362 works (i.e. Microsoft.Identity.Web 0.4.0)

So I looked in the object explorer for the method that was giving me the issue (`AddMicrosoftIdentityWebApi`), and OE was telling me that it was from a library based on .NET Core 2.2. So it's probably referring to an old package that I made myself a while ago which I had purged from my own NuGet server and pretty much all of my code.

To fix:
- Deleted any folders referring to this package in C:\Users\\{me}\\.nuget\packages
- Did a super-clean build (i.e. manually deleted build folders)

And it works.

And that's what I get for hi-jacking someone else's namespace.
